File: BackEndAPI.Java/DiChoThue/src/main/java/com/example/DiChoThue/Entities/DanhGia.java
package com.example.DiChoThue.Entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "DANH_GIA")
public class DanhGia {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int ma_danh_gia;
    private int ma_nguoi_dung;
    private int ma_chi_tiet_don_hang;
    private int ma_san_pham;
    private int sao;
    private String binh_luan;
}

File: BackEndAPI/Data/Entities/DanhGia.cs
using BackEndAPI.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BackEndAPI.Data.Entities
{
    public class DanhGia
    {
        public int MaDanhGia { get; set; }
        public int MaNguoiDung { get; set; }
        public int MaChiTietDonHang { get; set; }
        public int MaSanPham { get; set; }
        public int Sao { get; set; }
        public string BinhLuan { get; set; }
        public NguoiDung NguoiDung { get; set; }
        public ChiTietDonHang ChiTietDonHang { get; set; }
        public SanPham SanPham { get; set; }
    }
}
